### PR TITLE
Require 2 unit admins and recommend 3

### PR DESCRIPTION
**Pull request by inaod568**
**Test with:** https://github.com/ScilifelabDataCentre/dds_cli/pull/403

- [x] Tests passing
- [x] Black formatting
- [ - ] Migrations for any changes to the database schema
- [x] Rebase/merge the `dev` branch
- [x] Note in the CHANGELOG

### CODE REVIEW

**Comment by inaod568** [comment]
The tests will definitely fail. I'm fixing. When reviewing, focus on the code and testing and ignore the tests for now.

File: dds_web/api/project.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ####################################################################################################
 
 # Standard Library
+from email import message
```
Comment (aanil) [inline dds_web/api/project.py:8]: This import does not seem to be used.

File: tests/test_project_creation.py [hunk 2]
```diff
@@ -4,8 +4,11 @@
 import http
 import datetime
 import json
+from tokenize import Triple
+from turtle import update
```
Comment (aanil) [inline tests/test_project_creation.py:8]: These imports don't seem to be used.

File: dds_web/api/project.py (hunk 1 above)
Comment (inaod568) [reply dds_web/api/project.py:8]:
```suggestion
```

File: tests/test_project_creation.py (hunk 2 above)
Comment (inaod568) [inline tests/test_project_creation.py:8]:
```suggestion
```

File: tests/test_project_status.py [hunk 3]
```diff
@@ -88,6 +100,12 @@ def test_submit_request_with_invalid_args(module_client, test_project):
 
 def test_set_project_to_deleted_from_in_progress(module_client, boto3_session):
     """Create project and set status to deleted"""
+    current_unit_admins = models.UnitUser.query.filter_by(unit_id=1, is_admin=True).count()
+    assert current_unit_admins == 3
+    # create_unit_admins(num_admins=2)
+
+    # current_unit_admins = models.UnitUser.query.filter_by(unit_id=1, is_admin=True).count()
+    # assert current_unit_admins == 3
```
Comment (aanil) [inline tests/test_project_status.py:108]: Maybe remove this if its not needed?

**Review by aanil — APPROVED** [review]
Looks ok to me!

File: tests/test_project_status.py (hunk 3 above)
Comment (inaod568) [reply tests/test_project_status.py:108]:
```suggestion
```

**Pull request merged**